docs/src/modules/components/ThemeContext.js

```diff
@@ -48,12 +48,32 @@ export function Provider(props) {
         throw new Error(`Unrecognized type ${action.type}`);
     }
   }, themeInitialOptions);
-  const { direction, paletteColors, paletteType } = themeOptions;
+
+  const prefersDarkMode = useMediaQuery('@media (prefers-color-scheme: dark)');
```

Comment:
`@media` is not required (will be stripped in useMediaQuery) but it makes syntax errors more obvious: I used `useMediaQuery('prefers-color-scheme: dark');` before. Kudos to @oliviertassinari for spotting this.